### ISSUE TITLE
No GitHub Wiki

### ISSUE THREAD

**Issue by liam4** [jmoenig/Snap--Build-Your-Own-Blocks#901]
That stops me - and anybody else - from working on a wiki. :P

**Comment by cycomachead** [comment]
What would be the goals of the Wiki?

Also is github really the best place for a wiki? I guess that depends on how we want to use it... 
A "help" style wiki could be nice, but I'm not so sure that belongs on Github.

**Comment by brianharvey** [comment]
Yes, Liam, say more about what sort of wiki you have in mind.  But bear in mind that eventually we'll have our own Scratch-like web site supporting forum, wiki, and all that.

**Comment by bb010g** [comment]
I'd recommend setting up some sort of external wiki ([DokuWiki](https://www.dokuwiki.org/) is my recommendation), as while you can make [great GitHub wikis](https://github.com/showcases/projects-with-great-wikis), you're limited by it being static on GitHub. A normal wiki engine has a lot more flexibility.

**Comment by cycomachead** [comment]
I think the thing is -- we're (semi-) actively working on a more interactive snap.berkeley site, so it'd be nice if everything were integrated. But I guess it all depends on times and use cases.

I could see a developer facing wiki on github being useful.

**Comment by liam4** [comment]
The wiki would be a place to put tutorialson various topics, as well as explain how some of the blocks work, especially the ones that people from Scratch likely won't understand.

Also.. where's the GitHub repo for the Snap Site 3? :P

**Comment by brianharvey** [comment]
Ha ha.  Btw when are you coming to a Scratch conference or something so we can talk f2f?

**Comment by liam4** [comment]
Nope, I'm not.. ;-;

Unless there happens to be one some place in eastern Canada.. I don't think so. :P One day!

> Also.. where's the GitHub repo for the Snap Site 3? :P

**Comment by brianharvey** [comment]
Do you mean the repo for BYOB3?  There isn't any.  Strange as it may seem, Jens made it without git.

Eastern Canada is basically Boston.  Come next summer!

**Comment by liam4** [comment]
No, I mean the repo for http://snap.berkeley.edu.

Maybe.. ;)

**Comment by brianharvey** [comment]
Oh.  Sorry... no repo.  We're not expecting a lot of development on this site, since we're hoping it will pretty soon be obsolete, replaced by a full Scratch-like site.  (Sorry.)

**Comment by liam4** [comment]
The repo for the future Snap! site*
